packages/backend/server/package.json

```diff
@@ -15,6 +15,8 @@
     "test:copilot": "ava \"src/__tests__/**/copilot-*.spec.ts\"",
     "test:coverage": "c8 ava --concurrency 1 --serial",
     "test:copilot:coverage": "c8 ava --timeout=5m \"src/__tests__/**/copilot-*.spec.ts\"",
+    "e2e": "cross-env TEST_MODE=e2e ava",
+    "e2e:coverage": "cross-env TEST_MODE=e2e c8 ava",
```

Comment:
Need a new GitHub Action job to run `yarn affine @affine/server e2e:coverage --forbid-only`